.fixtures.yml

```diff
@@ -3,4 +3,4 @@
 ---
 fixtures:
   forge_modules:
-#     stdlib: "puppetlabs/stdlib"
+    stdlib: "puppetlabs/stdlib"
```

Comment:
We opted to skip the stdlib dependency and just copy the types over.

Reply:
This implementation uses `join()`, so has a dependency on either stdlib or on Puppet 5.5. Given that it has a dependency on stdlib anyway I figured we might as well use the standard types rather than copying them.

If you prefer, I can rework this to express a dependency on an older stdlib. IMO that's not as clean a solution though.